Tests/BUILD

```diff
@@ -0,0 +1,66 @@
+load("@build_bazel_rules_apple//apple:macos.bzl", "macos_build_test")
+load("@build_bazel_rules_apple//apple:watchos.bzl", "watchos_build_test")
+load("@build_bazel_rules_apple//apple:tvos.bzl", "tvos_build_test")
+load("@build_bazel_rules_apple//apple:ios.bzl", "ios_build_test")
+load("@build_bazel_rules_swift//swift:swift.bzl", "swift_test")
+
+config_setting(
+    name = "linux",
+    constraint_values = ["@platforms//os:linux"],
+)
+
+genrule(
+    name = "LinuxMain",
+    srcs = ["LinuxMain.swift"],
+    outs = ["main.swift"],
+    cmd = "cp $< $@"
```

Comment:
Nice trick. Although I suppose it’d be even better if the Swift bazel rules supported `LinuxMain.swift` for parity with SwiftPM.

Reply:
Yea its a bummer I need to do this at all, its the cleanest option I saw. @Keith have you seen this limitation in `swift_test` before?

Reply:
hrm we include a generated LinuxMain.swift directly, were you hitting a filename issue or what?

Reply:
It never worked unless it was called `main.swift`, I got invalid top level declaration errors